apps/breethe/lib/breethe/sources/eea.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Breethe.Sources.EEA do
+  alias __MODULE__.CSV
+
+  # @parameters ["PM10", "PM2.5", "SO2", "NO2", "O3", "CO"]
+  @parameters ["SO2", "NO2", "O3", "CO"]
+
+  def country_codes(),
```

Comment:
I'd change this to use the same pattern as for `@parrameters` with `def country_codes(), do: @country_codes` for consistency.

Reply:
fixed